dashboard/app/models/experiment.rb

```diff
@@ -0,0 +1,74 @@
+# == Schema Information
+#
+# Table name: experiments
+#
+#  id                     :integer          not null, primary key
+#  name                   :string(255)
+#  type                   :string(255)
+#  start_time             :datetime
+#  end_time               :datetime
+#  created_at             :datetime         not null
+#  updated_at             :datetime         not null
+#  percentage             :integer
+#  earliest_section_start :datetime
+#  latest_section_start   :datetime
+#  section_id             :integer
+#
+
+MAX_CACHE_AGE = 1.minutes
+
+class Experiment < ApplicationRecord
+  validates :name, presence: true
```

Comment:
Is there a reason to validate this at the rails level but not enforce it at the DB level?